enterprise/templates/enterprise/emails/user_notification.txt

```diff
@@ -0,0 +1,9 @@
+{% if user_name %}Dear {{ user_name }},{% else %}Hi!{% endif %}
+
+You have been enrolled in {{ course_name }}, a {% if program_type %}{{ program_type }} {% endif %}{{ course_type }} offered by {{ organization_name }}. This {{ course_type }} begins {{ start_date | date:"F j, Y" }}. For more information, see the following link:
```

Comment:
Can you mark this up for i18n?

Also, this way of combining integrating `course_type` and other vars into a sentence does not work for translation, even if the word `"course"` is translated, because it may need to be inflected differently depending on the context of the translated sentence. So you'll need to refactor this a bit.

```
{%if is_a_program%}
{% blocktrans %}
You've been enrolled in the {{name}} {{program_type}} program, ...
{% blocktrans %}
{%else%}
...
{%endif%}
```

(`program_type` can be inlined since it's a trademarked name and would not be translated nor inflected.)